source/fundamentals/collations.txt

```diff
@@ -4,5 +4,311 @@ Collations
+   côté
+
+When using the Canadian French collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   côte
+   coté
+   côté
+
+Specify a Collation
+-------------------
+
+To specify a collation, create a ``Collation`` object. You must define the ``Locale`` field
+of the ``Collation`` object; all other fields are optional. For example, the following code
+snippet specifies a ``Collation`` object with the ``"en_US"`` locale collation:
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "en_US"}
+
+For a complete list of ``Collation`` object fields, visit the `Collation API documentation 
+<{+api+}/mongo/options#Collation>`__. To see all the supported locales and the
+default values for the ``Locale`` fields, visit :manual:`Supported Languages and Locales
+</reference/collation-locales-defaults/#supported-languages-and-locales>`.
+
+Usage
+-----
+
+For information on applying a collation to a collection, view, index, or operation, navigate
+to the following sections:
+
+- :ref:`golang-collection-collation`
+- :ref:`golang-index-collation`
+- :ref:`golang-op-collation`
+
+.. _golang-collection-collation:
+
+Set a Collation on a Collection or View
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can apply a collation when you create a new collection or view. This defines the default
+collation for any operations called on that collection or view. Set a collation through a 
+``CreateCollectionOptions`` or ``CreateViewOptions`` object. Then, call the
+``CreateCollection()`` or ``CreateView()`` method with your options object as an argument.
+
+.. _golang-create-collection:
+
+Create a Collection Example
+```````````````````````````
+
+The following example creates a new collection called ``books`` and specifies a default
+collation with the ``"fr"`` locale. The ``Strength`` collation field has a value of ``1``
+to ignore differences in letter accents.
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "fr", Strength: 1}
+   opts := options.CreateCollection().SetCollation(myCollation)
+   err := db.CreateCollection(context.TODO(), "books", opts)
+
+   if err != nil {
+      log.Fatal(err)
+   }
+
+Use the Default Collation Example
+`````````````````````````````````
+
+If you call an operation that supports collation on the ``books`` collection, the operation
+will now use the default collation.
+
+Assume the ``books`` collection contains the following documents:
+
+.. code-block:: json
+
+   {"name" : "Emma", "length" : "474"}
+   {"name" : "Les Misérables", "length": "1462"}
+   {"name" : "Infinite Jest", "length" : "1104"}
+   {"name" : "Cryptonomicon", "length" : "918"}
+   {"name" : "Ça", "length" : "1138"}
+
+.. note::
+
+   To learn how to insert documents, see :ref:`golang-insert-guide`.
+   
+You can use an operation with the default collation on the ``books`` collection specified
+in the :ref:`golang-create-collection`. The following example uses the ``Find()`` method
```

Comment:
Issue/Question: Does this sentence add anything not covered by the first sentence in this section? My guess is no, but I'm not sure.

```suggestion
The following example uses the ``Find()`` method
```

Reply:
I deleted this sentence and linked the section where the default collation is set in the first sentence of "Use the Default Collation Example"